content/concepts/consensus-network/consensus-protections.md

```diff
@@ -0,0 +1,67 @@
+# Consensus Protections Against Attacks and Failure Modes
+
+The XRP Ledger Consensus Protocol is a _byzantine fault tolerant_ consensus mechanism, which means it's designed to operate even if all kinds of things can go wrong: participants depend on an unreliable open network to communicate, and malicious actors may be attempting to control or interrupt the system at any given time. On top of that, the set of participants in the XRP Ledger Consensus Protocol isn't known in advance and can change over time.
+
+Confirming transactions in a timely manner while maintaining [the desired properties of the network](intro-to-consensus.html#consensus-protocol-properties) is a complex challenge, and it's impossible to build a perfect system. The XRP Ledger Consensus Protocol is designed to work as well as it can in most situations, and to fail as gracefully as possible in the situations where it can't.
+
+This page describes some of the types of challenges that the XRP Ledger Consensus Protocol faces and how it handles them.
+
+## Individual Validators Misbehaving
+
+_Validators_ are servers who actively contribute to the process of deciding each new ledger version. Validators only have an influence over servers configured to trust them (including indirectly). Consensus can continue even if some validators are misbehaving, including a large variety of failure cases, such as:
```

Comment:
Perhaps: 

```
< Validators are servers who
---
> Validators are servers that

```
Per the Chicago Manual of Style (CMOS) 5.56: _Who normally refers to a human being, but it can be used in the first, second, or third person._